index.html

```diff
@@ -745,19 +745,31 @@ <h2>
         A <dfn>navigation scope</dfn> is a [[!URL]] that represents the set of
         URLs to which an <a>application context</a> can be navigated while the
         manifest is <a>applied</a>. The <a>navigation scope</a> of a manifest
-        <var>manifest</var> is <var>manifest</var>["<a>scope</a>"]. If this is
-        <code>undefined</code> (due to an error, or because it was not declared
-        in the manifest), the navigation scope is <dfn>unbounded</dfn>.
+        <var>manifest</var> is <var>manifest</var>["<a>scope</a>"].
       </p>
+      <div class="note" data-link-for="WebAppManifest">
+        <p>
+          If the <a>scope</a> member is not present in the manifest, it
+          defaults to the directory containing the <a>start_url</a> member. For
```

Comment:
The problem is that there is not such thing as a "directory" in URLs (I know you mean path segments here)... it's still somewhat problematic, because the path segments are kinda meaningless or used to mean different things (unlike file system directories).

Reply:
I know, I was trying to avoid using the word "directory" but I just ran with it. Only in non-normative text. I can change it but do you have any suggested wording?

> it's still somewhat problematic

What do you mean? The wording is problematic, or the concept? I think the concept of cutting off the last part of the path is very well defined on the web (it's the standard URL resolution algorithm). The *reason* resolution works this way is because it allows HTML pages to reference a file in the same directory just by name, which is why I described it this way.

Reply:
>  I can change it but do you have any suggested wording?

We should follow terminology from the URL spec (i.e., path segment). 
 
> What do you mean? The wording is problematic, or the concept? 

Apologies, I accidentally missed this line: 
https://github.com/w3c/manifest/pull/647/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR1742

Which addresses the issue. There too, we might just point to the URL spec "single-dot path segment".

Reply:
OK. I didn't want to s/directory/path segment, because a path segment is just one piece, not the whole prefix. (For example, if the path is `/x/y/z.html`, the scope is `/x/y`, not just `y`.)

I've reworded it to refer to path terminology, not directory.

> There too, we might just point to the URL spec "single-dot path segment".

Not sure what you mean. Do you think the actual algorithm should say "single-dot path segment" instead of "."? I don't think so, because "." is a specific string, while "single-dot path segment" is a pattern match.